pennylane/transforms/core/transform_dispatcher.py

```diff
@@ -585,7 +586,10 @@ def _create_transform_primitive(name):
     def _(
         *all_args, inner_jaxpr, args_slice, consts_slice, targs_slice, tkwargs
     ):  # pylint: disable=unused-argument
-        raise NotImplementedError
+        args = all_args[args_slice]
+        consts = all_args[consts_slice]
+
+        return qml.capture.eval_jaxpr(inner_jaxpr, consts, *args)
```

Comment:
This was needed so that calling a transformed function does not raise a `NotImplementedError`.